.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: csharp
+
+os: linux
+dist: trusty
+sudo: required
+
+services:
+  - docker
+
+branches:
+  only: 
+    - master
+
+install:
+  - docker pull syncromatics/build-box:feature_add-xamarin-android-support
```

Comment:
What will the "proper" tag be for this image? `syncromatics/build-box:release-5.4`?